packages/env-spec-parser/src/classes.ts

```diff
@@ -299,26 +300,32 @@ export class ParsedEnvSpecBlankLine {
 
 
 export class ParsedEnvSpecConfigItem {
-  expandedValue: ParsedEnvSpecStaticValue | ParsedEnvSpecFunctionCall | undefined;
+  value: ParsedEnvSpecStaticValue | ParsedEnvSpecFunctionCall | undefined;
 
   constructor(public data: {
     key: string;
     value: ParsedEnvSpecStaticValue | ParsedEnvSpecFunctionCall | undefined;
     preComments: Array<ParsedEnvSpecDecoratorComment | ParsedEnvSpecComment>;
     postComment: ParsedEnvSpecDecoratorComment | ParsedEnvSpecComment | undefined;
     _location?: any;
-  }) {}
+  }) {
+    if (this.data.value) {
+      const expanded = expand(this.data.value!);
```

Comment:
expansion now happens automatically within the parser library, rather than needing to be called explicitly